packages/ui-components/src/components/assistant-ui/history/assistant-ui-history-item.tsx

```diff
@@ -84,37 +84,41 @@ const AssistantUiHistoryItem = ({
             onChange={(e) => setEditedName(e.target.value)}
             onKeyDown={handleKeyDown}
             onBlur={handleRename}
-            className="flex-1 bg-transparent border-none outline-none focus:ring-0 font-normal dark:text-primary text-sm leading-snug"
+            className="flex-1 bg-transparent border-none outline-none focus:ring-0 font-normal text-black text-[14px] leading-[20px]"
```

Comment:
text-sm is 14px and leading 20

```suggestion
            className="flex-1 bg-transparent border-none outline-none focus:ring-0 font-normal text-black text-sm"
```

Reply:
Done